### PR TITLE
grafana(dm): fix grafana expr

### LINKED ISSUE #4404: dashboard: `binlog file gap between relay` have no data
### What did you do?

see this dashboard in grafana

### What did you expect to see?

see normal data

### What did you see instead?

middle_img_v2_1e78e593-57c8-494f-b292-e6b5d4bcc6cg

### Versions of the cluster

DM version (run `dmctl -V` or `dm-worker -V` or `dm-master -V`):
```console
(paste DM version here, and you must ensure versions of dmctl, DM-worker and DM-master are same)
```
Upstream MySQL/MariaDB server version:
```console
(paste upstream MySQL/MariaDB server version here)
```
Downstream TiDB cluster version (execute `SELECT tidb_version();` in a MySQL client):
```console
(paste TiDB cluster version here)
```
How did you deploy DM: tiup or manually?
```console
(leave TiUP or manually here)
```
Other interesting information (system version, hardware config, etc):
```console
>
>
```

### current status of DM cluster (execute `query-status <task-name>` in dmctl)

```console
(paste current status of DM cluster here)
```

### PR DESCRIPTION
**Pull request by Ehco1996**
### What problem does this PR solve?

Issue Number: close #4404 

### What is changed and how it works?

use right expr

* fix standard's lag lable not show
stander lag label

* fix binlog file gap between master and syncer
截屏2022-01-19 下午3 04 15

* fix binlog file gap between relay and syncer
截屏2022-01-19 下午3 01 32

* fix binlog file gap between master and relay
截屏2022-01-19 下午3 05 00

### Check List 

Tests 

 - Manual test (add detailed scripts or steps below)
 - No code

Code changes

Side effects

Related changes

 - Need to cherry-pick to the release branch

### Release note 

```release-note
`None`.
```

### CODE REVIEW

File: dm/dm/dm-ansible/scripts/DM-Monitor-Professional.json [hunk 1]
```diff
@@ -3716,12 +3716,12 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "dm_relay_binlog_file{instance=~\"$instance\", node=\"relay\"} - ON(instance, task) dm_syncer_binlog_file{instance=~\"$instance\", task=~\"$task\", node=\"syncer\"}",
+          "expr": "dm_relay_binlog_file{instance=~\"$instance\", node=\"relay\"} - ON(instance) dm_syncer_binlog_file{instance=~\"$instance\", task=~\"$task\", node=\"syncer\"}",
           "format": "time_series",
           "hide": false,
           "interval": "$interval",
           "intervalFactor": 2,
-          "legendFormat": "{{task}} - {{instance}}",
```
Comment (glorv) [inline dm/dm/dm-ansible/scripts/DM-Monitor-Professional.json:3724]: I think we should remain `{{task}} - {{instance}}` as each task will produce a different value

Comment (Ehco1996) [reply dm/dm/dm-ansible/scripts/DM-Monitor-Professional.json:3724]: this metric: `dm_relay_binlog_file`  don't have label `task` so this legend looks like this ` - 127.0.0.1:8621`

```


dm_relay_binlog_file{instance="127.0.0.1:8262", job="dm-worker-1", node="master"} | 1
-- | --
dm_relay_binlog_file{instance="127.0.0.1:8262", job="dm-worker-1", node="relay"}


```

Comment (Ehco1996) [reply dm/dm/dm-ansible/scripts/DM-Monitor-Professional.json:3724]: ping @glorv  ptal

**Comment by Ehco1996** [comment]
could you please take another look for this pr ? thanks 🧡🧡🧡 

/cc @lance6716 @glorv

File: dm/dm/dm-ansible/scripts/DM-Monitor-Professional.json [hunk 2]
```diff
@@ -3716,12 +3716,12 @@
...
+          "legendFormat": "{{{instance}}",
```
Comment (lance6716) [inline dm/dm/dm-ansible/scripts/DM-Monitor-Professional.json:3724]:
```suggestion
          "legendFormat": "{{instance}}",
```

could you provide an environment to let me verify the layout?

**Review by lance6716 — CHANGES_REQUESTED** [review]

File: dm/dm/dm-ansible/scripts/DM-Monitor-Professional.json [hunk 3]
```diff
@@ -3716,12 +3716,12 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "dm_relay_binlog_file{instance=~\"$instance\", node=\"relay\"} - ON(instance, task) dm_syncer_binlog_file{instance=~\"$instance\", task=~\"$task\", node=\"syncer\"}",
+          "expr": "dm_relay_binlog_file{instance=~\"$instance\", node=\"relay\"} - ON(instance) dm_syncer_binlog_file{instance=~\"$instance\", task=~\"$task\", node=\"syncer\"}",
```
Comment (Ehco1996) [inline dm/dm/dm-ansible/scripts/DM-Monitor-Professional.json:3719]: preview [link](http://172.16.5.177:3000/d/Ja68YqRnzxxxxxxx/dm-monitor-professional-fixed?editPanel=44&orgId=1&from=now-5m&to=now&var-task=All&var-source=mysql-replica-2-001&var-instance=172.16.5.177:8101&var-queueNo=All&var-type=All&var-stage=All&var-target_schema=All&var-target_table=All&var-interval=$__auto_interval_interval&var-worker=All)

File: dm/dm/dm-ansible/scripts/DM-Monitor-Professional.json [hunk 4]
```diff
@@ -5088,7 +5088,7 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "dm_relay_binlog_file{instance=~\"$instance\", node=\"master\"} - ON(instance, job) dm_relay_binlog_file{instance=~\"$instance\", node=\"relay\"}",
+          "expr": "dm_relay_binlog_file{instance=~\"$instance\", node=\"master\"} - ON(instance) dm_relay_binlog_file{instance=~\"$instance\", node=\"relay\"}",
```
Comment (Ehco1996) [inline dm/dm/dm-ansible/scripts/DM-Monitor-Professional.json:5091]: preview [link](http://172.16.5.177:3000/d/Ja68YqRnzxxxxxxx/dm-monitor-professional-fixed?editPanel=42&orgId=1&from=now-5m&to=now&var-task=All&var-source=mysql-replica-2-001&var-instance=172.16.5.177:8101&var-queueNo=All&var-type=All&var-stage=All&var-target_schema=All&var-target_table=All&var-interval=$__auto_interval_interval&var-worker=All)

File: dm/dm/dm-ansible/scripts/DM-Monitor-Standard.json [hunk 5]
```diff
@@ -450,23 +450,23 @@
       "targets": [
         {
           "exemplar": true,
-          "expr": "histogram_quantile(0.90, sum(rate(dm_syncer_replication_lag_bucket{instance=~\"$instance\",task=~\"$task\"}[$interval])) by (le,instance,task))",
+          "expr": "histogram_quantile(0.90, sum(rate(dm_syncer_replication_lag_bucket{source_id=~\"$source\",task=~\"$task\"}[$interval])) by (le,task,source_id))",
```
Comment (Ehco1996) [inline dm/dm/dm-ansible/scripts/DM-Monitor-Standard.json:453]: preview [link](http://172.16.5.177:3000/d/canCEQgnkxxadasd/dm-monitor-standard-fixed?viewPanel=38&orgId=1&refresh=5s)

File: dm/dm/dm-ansible/scripts/DM-Monitor-Professional.json (hunk 2 above)
Comment (Ehco1996) [reply dm/dm/dm-ansible/scripts/DM-Monitor-Professional.json]: thanks! my bad , add preview link in about review comments now

File: dm/dm/dm-ansible/scripts/DM-Monitor-Professional.json (hunk 1 above)
Comment (lance6716) [reply dm/dm/dm-ansible/scripts/DM-Monitor-Professional.json:3724]: maybe you can use `group_right`

```
dm_relay_binlog_file{instance="172.16.5.177:8101", node="relay"} - ON(instance) group_right dm_syncer_binlog_file{instance="172.16.5.177:8101", node="syncer"}
```

please find a good solution from https://prometheus.io/docs/prometheus/latest/querying/operators/#many-to-one-and-one-to-many-vector-matches

Comment (Ehco1996) [reply dm/dm/dm-ansible/scripts/DM-Monitor-Professional.json:3724]: due to only syncer_binlog_fie have many to on relationship with relay_binlog_file, only one dashboard need to be adjusted. added in https://github.com/pingcap/tiflow/pull/4405/commits/28f599425c64fd2c4fbd88995c324dd4dfecd228 , preview link is [here](http://172.16.5.177:3000/d/Ja68YqRnzxxxxxxx/dm-monitor-professional-fixed?editPanel=44&orgId=1)

**Review by lance6716 — APPROVED** [review]

**Review by glorv — APPROVED** [review]

**Pull request merged**